tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -20,6 +20,21 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def ping_via_console(src_vm, dst_vm):
+    """
+    Ping from src_vm to dst_vm via VM console to avoid SSH masking issues.
+    Uses '-c 10 -w 10' so that any packet loss causes a non‑zero exit status.
+    """
+    dst_ip = dst_vm.vmi.interfaces[0]["ipAddress"]
+    console_command_timeout_seconds = 10
```

Comment:
no reason to save and then use

Reply:
> The only missing info is the "sec", however, by default most (if not all) timeouts are set with that scale.

@EdDev You've been deep in python/Go and away from C for quite too long...